docs/Advanced-Topics.rst

```diff
@@ -77,3 +77,26 @@ Recommendations for gcc Users (MinGW, \*nix)
 --------------------------------------------
 
 -  Refer to `gcc Tips <./gcc-Tips.rst>`__.
+
+Support for Position Bias Treatment
+--------------------------------------------
+
+Often the relevance labels provided in Learning-to-Rank taks might be derived from implicit user feedback (e.g., clicks) and therefore might be biased due to their position on the screen when having been presented to a user. LightGBM uses an additional file to store positional data, like the following:
+
+::
+
+    3
+    4
+    1
+    ...
+
+It means the position of the document from the first data row, when presented to a user, was ``3``, second was ``4``, and so on. A position could be any (arbitrary) integer number and could encode the presentation of an item even in complex layouts (it is up to the LightGBM user to perform such encoding/enumeration).
```

Comment:
```suggestion
It means the document from the first data row was the third presented to a user, the document from the second row was the fourth presented to the user, and so on. A position could be any (arbitrary) integer number and could encode the presentation of an item even in complex layouts (it is up to the LightGBM user to perform such encoding/enumeration).
```

Is this index 0-based (first position = `0`) or 1-based (first position = `1`)? Can you please clarify that?

Reply:
@jameslamb, the positions don't necessarily have to be ordinal or coming from a contiguous range. So if we require them to be integers (not strings), they could be any integer (I guess even negative). As such, there may not necessarily exist entities defined as "first position" or "second position" (e.g., in complex layouts).